renew-le.sh

```diff
@@ -5,14 +5,14 @@ WORKDIR=$(dirname "$(realpath $0)")
 EMAIL=""
 
 ### cron
-# check that the cert will last at least 2 days from now to prevent too frequent renewal
+# check that the cert will last at least 30 days from now to prevent too frequent renewal
```

Comment:
I think this comment needs a re-write. It will trigger a renewal 30 days prior to expiration (or whatever value we come up with).